src/Apps/Artwork/artworkRoutes.tsx

```diff
@@ -62,12 +62,12 @@ export const artworkRoutes: RouteProps[] = [
         }
       }
     `,
-    getCacheConfig: ({ context }) => {
-      return {
-        force: true,
+    getCacheConfig: ({ context, ...rest }) => {
+      const isLoggedIn = !!context.user
 
-        // TODO: In the future, we might want to only cache for logged out users
-        // force: !!context.user,
+      return {
+        // Fetch artwork if logged in, otherwise cache
+        force: isLoggedIn,
```

Comment:
Since we always read `serverCacheTTL` during the cache write process, we can easily make this more dynamic depending on the type of artwork, etc. Something to think about for the future.